test/assetcompute.test.js

```diff
@@ -102,8 +259,35 @@ describe( 'assetcompute.js tests', () => {
         }
     });
 
-    it('should fail calling /process', async function() {
-        const { AssetCompute } = require('../lib/assetcompute');
+    it('should fail with asset compute /unregister hits 429', async function() {
+        const options = {
+            accessToken: 'accessToken',
+            org: 'org',
+            apiKey: 'apiKey'
+
+        };
+        nock('https://asset-compute.adobe.io')
+            .post('/unregister')
+            .reply(429,{
+                ok: false,
+                error_code:'429050',
+                message:'Too many requests'
+            }, {
+                'retry-after': 3
+            });
+
+        const assetCompute = new AssetCompute(options);
+        try {
+            await assetCompute.unregister();
+        } catch (error) {
+            assert.strictEqual(error.code, 429);
+            assert.strictEqual(error.name, 'TooManyRequestsError');
+            assert.strictEqual(error.message, 'Unable to invoke /unregister: 429: {"ok":false,"error_code":"429050","message":"Too many requests"} (details: {"size":0,"timeout":30000})');
```

Comment:
curious, where is `30000` coming from ?

Reply:
this is taken from real-world example of trying this out in IT tests and seeing this message. I am not totally sure as the api-gateway docs do not provide much details: https://wiki.corp.adobe.com/display/API/Error+Codes+Documentation+-+API+Gateway